detector/db.go

```diff
@@ -1,4 +1,4 @@
-package ld
+package detector
```

Comment:
maybe is more meaningful call the package `language`

Reply:
@mcuadros did you mean `license`?

Reply:
what about ... `licensedb`? and we change the name of the whole project to that?

Reply:
Why db? The data is taken from SPDX; this seems misleading. `go-license-detector` is great: it shows exactly what the project does.